.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+language: go
+go:
+- 1.9
+services:
+- docker
+jobs:
+  include:
+  - stage: Check generated contents are up to date and code is formatted.
+    script: ./scripts/jenkins/check-make-generate.sh
+  - stage: Compile binary
+    script: make crossbuild
```

Comment:
To save execution time either this or https://github.com/coreos/prometheus-operator/blob/master/scripts/jenkins/run-e2e-tests.sh#L23 can be skipped.